src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -49,17 +61,17 @@ public function normalize($object, $format = null, array $context = array())
         }
 
         $result = array(
-            'type' => $context['type'] ?? 'https://symfony.com/errors/validation',
-            'title' => $context['title'] ?? 'Validation Failed',
+            'type' => $context[static::TYPE] ?? $this->defaultContext[static::TYPE] ?? 'https://symfony.com/errors/validation',
```

Comment:
self:: instead of static::?
generally we should use self:: when using constants when possible IMHO